### DISCUSSION TITLE
Optimal settings for always plugged in Android alarm clock?

### DISCUSSION THREAD

**Discussion by Ciantic** [VR-25/acc#168] [Q&A]
Hi!

I'm trying to turn old Android device (Moto G5) to always plugged in alarm clock / assistant.

I don't know much about the settings, but I think I want something like this:

Always on idle mode, unless battery has drained to 20%, then charge back to 60% and continue in idle mode.

Basically it should sit weeks (if not months?) in idle mode because always plugged in alarm clock should not drain battery. It should drain battery only when there is power outage.

I just installed ACC, and exploring the settings, there is a lot! Any help is appreciated, thank you!

**Comment by Bryantdl7** [comment]
I run stop at 75%, idle mode on, and resume charging at 70%. I use a Moto G7 Power as my alarm clock phone. That way I have enough battery if the power goes out, but I am also not stressing the battery much. 

In my idle mode setup, when I install the AccA app, I see the battery is drawing 0mA from the battery when plugged in and between 71-75%, that is another factor which will help protect your battery too, if its not getting used when on idle mode, as designed.

**Comment by VR-25** [comment]
`acc -s pc=60 mcc=500 mcv=3920`
